kubernetes/config-maps/nginx-proxy/configs/http.conf.j2

```diff
@@ -20,15 +20,19 @@ upstream {{ current_golem_messages_version }} {
      server nginx-proxy.default.svc.cluster.local;
 }
 
+{% if previous_golem_messages_version is not none and previous_golem_messages_version is defined %}
```

Comment:
You should assume that every config variable is always defined. It may be empty but not undefined.